crmd/lrm.c

```diff
@@ -1823,6 +1832,10 @@ construct_op(lrm_state_t * lrm_state, xmlNode * rsc_op, const char *rsc_id, cons
 
     params = xml2list(rsc_op);
     g_hash_table_remove(params, CRM_META "_op_target_rc");
+    
+    if (!is_remote_lrmd_ra(NULL, NULL, rsc_id)) {
+        op->versioned_params = first_named_child(rsc_op, XML_TAG_VER_ATTRS);
```

Comment:
We only need to add versioned parameters to params hash table in construct_op(). We don't actually use this pointer later. But I think you're right, it's the proper thing to do.